docs/reference/migration/migrate_8_0/scripting.asciidoc

```diff
@@ -0,0 +1,23 @@
+[discrete]
+[[breaking_80_scripting_changes]]
+==== Scripting changes
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+.JodaCompatibleDateTime is removed from scripting.
```

Comment:
Minor nit: `has been removed` is a little more consistent with our other deprecations.

Feel free to ignore if wanted.
```suggestion
.The `JodaCompatibleDateTime` class has been removed.
```

Reply:
Fixed locally.